cmake/Modules/opm-material-prereqs.cmake

```diff
@@ -16,8 +16,8 @@ set (opm-material_DEPS
 	# compile with C++0x/11 support if available
 	"CXX11Features REQUIRED"
 	# prerequisite OPM modules
-	"opm-common"
 	"opm-parser"
+	"opm-common"
```

Comment:
no, only opm-common should be marked as REQUIRED, opm-parser is still optional (and as far as I'm concerned, it will stay that way in the foreseeable future). also, only the build system and some pretty basic headers (e.g., exceptions) are used from opm-common in opm-material, so it would be nice if opm-common could be used without introducing any additional runtime dependency (which is dormant in the case of opm-material without opm-parser). I suppose that's a wish I should send to Santa, though.